go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-datadog
 
 require (
-	github.com/DataDog/datadog-api-client-go v1.6.0
+	github.com/DataDog/datadog-api-client-go v1.6.1-0.20211111153236-e7664265bffa
```

Comment:
Commit is too old, can you upgrade go client again ?

Reply:
@zippolyte I've gone ahead and updated the go client.